pkg/api/authz/authz.go

```diff
@@ -104,6 +104,44 @@ var staticRules = map[string][]string{
 		"GET /api/all-mcps/servers",
 		"GET /api/all-mcps/servers/{mcp_server_id}",
 	},
+	PowerUserGroup: {
+		// Workspace-scoped MCP Server Catalog Entries
+		"GET /api/workspaces/{workspace_id}/entries",
```

Comment:
IIUC, this will allow _ANY_ user with the `PowerUserGroup` to access resources in _ALL_ workspace, not just the ones they own.

I had thought authz for this was handled by https://github.com/obot-platform/obot/blob/05ae9b4e7c556c74555e0c98fd74ba6f0bf312c5/pkg/api/authz/resources.go#L304